spec/features/card_pages/card_pages_spec.rb

```diff
@@ -86,6 +86,78 @@
           expect(Card.last.assignees.count).to eq 2
         end
       end
+
+      describe 'local caching' do
```

Comment:
@sean-yeoh  Can we move this into a shared examples file? The pattern seems to be repeated across multiple types of resources, so I believe it's possible to reduce the repetition.

For each resource, we can define a field (`evidence[content]` for evidence, `issue[text]` for issues, etc.) which we use to check for pre-filled data. The 3 cases that are shared are: resource is saved, resource is not saved, and cancel button is clicked. The other cases can be defined outside the shared space. Thoughts?

Reply:
@aapomm That sounds much better than the current way. Thanks for the suggestion! i will be updating that shortly